### ISSUE TITLE
Adding an autocmd TermClose in init.vim breaks the stabilization

### ISSUE THREAD

**Issue by pure-bliss** [luukvbaal/stabilize.nvim#18]
## Setup information

Using stabilize.nvim 174dfcd

```
~/.l/s/n/p/s/lua $ nvim --version 
NVIM v0.7.0
Build type: Release
LuaJIT 2.1.0-beta3
Compiled by builduser

Features: +acl +iconv +tui
See ":help feature-compile"

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "/usr/share/nvim"

Run :checkhealth for more info
```

Stabilize setup config:

```
require("stabilize").setup({
    ignore = {
		filetype = { "help", "list", "Trouble" },
		buftype = { "quickfix", "loclist" }
	},
	nested = nil
})

```

## Description

First of all I haven't tried this issue with a minimal init.vim, because the issue does not appear as soon as I remove `TermClose` autocmd from my init.vim.

Basicaly I use `autocmd TermClose * :q` to automatically close the terminal, because I don't want to see the 'process exited ...' text every time I exit the shell.

After messing around I realized that removing that autocmd makes everything work again.

Here is a video I recorded of the issue:

https://user-images.githubusercontent.com/83361786/174862590-e224c068-732b-4288-9ac6-61bf227a0d20.mp4

The smaller window is `:CocDiagnostics` mapped to `<C-i><C-d>,` the one with the shell is a terminal: `nnoremap <silent> <C-d> :sp\|te<CR>` As the video shows sometimes the stabilization works, sometimes not. After removing the `TermClose` autocmd everything works reliably.

Please let me know if some additional information is needed.

**Comment by luukvbaal** [comment]
Adding `nested = "TermClose"` to this plugin's config should fix the issue.

**Comment by pure-bliss** [comment]
I have tried that before, but thought that shouldn't be used there, because I get the following error on `TermClose`:

```
Error detected while processing TermClose Autocommands for "*":
Error executing lua callback: ...ocal/share/nvim/plugged/stabilize.nvim/lua/stabilize.lua:112: attempt to call field 'nvim_exec_autocmd' (a nil value)
stack traceback:
        ...ocal/share/nvim/plugged/stabilize.nvim/lua/stabilize.lua:112: in function <...ocal/share/nvim/plugged/stabilize.nvim/lua/stabilize.lua:111>
```
Setup:
```
require("stabilize").setup({
    ignore = {
		filetype = { "help", "list", "Trouble" },
		buftype = { "quickfix", "loclist" }
	},
	nested = "TermClose"
})
```

Just realized that `:help nvim_exec_autocmd` gives help about 'nvim_exec_autocmd*s*', and making these changes seem to fix the issue:

```
diff --git a/lua/stabilize.lua b/lua/stabilize.lua
index 86171a2..a7f9868 100644
--- a/lua/stabilize.lua
+++ b/lua/stabilize.lua
@@ -109,7 +109,7 @@ function M.setup(setup_cfg)
 
        if cfg.nested then
                api.nvim_create_autocmd(cfg.nested, { group = group, callback = function()
-                       api.nvim_exec_autocmd("User", { pattern = "StabilizeRestore" })
+                       api.nvim_exec_autocmds("User", { pattern = "StabilizeRestore" })
                end})
        end
 end
```

Looks like `nvim_exec_autocmd` got replaced with `nvim_exec_autocmds`

**Comment by pure-bliss** [comment]
The change I suggested fixes this issue, using branch 0.8 with neovim 0.8 works as expected, so I guess this can be closed.

Some distributions already provide 0.8 neovim, so the 100% solution would be checking which function to use, but that might be an overkill.
